services/QuillLMS/client/app/bundles/Shared/actions/questions.ts

```diff
@@ -0,0 +1,22 @@
+import { PRODUCTION } from "../../../constants/flagOptions";
+import { startListeningToQuestions as grammarListenToQuestions } from "../../Grammar/actions/questions";
+import connect from "../../Connect/actions/questions";
+import diagnostic from "../../Diagnostic/actions/questions";
+import { CONNECT, DIAGNOSTIC, GRAMMAR } from "../utils/constants";
+
+export const reloadQuestionsIfNecessary = (flag: String, activityType: String) => {
+  if (flag !== PRODUCTION) { return null; }
+  return (dispatch: Function) => {
+    switch (activityType) {
+      case GRAMMAR:
+        dispatch(grammarListenToQuestions());
```

Comment:
I created a shared action for this new method so that it could be used by all three modules. Emilia and I discussed passing the functions directly into the code rather than importing them which could also be done if folks have strong feelings. The advantage of this method is that it sets us up for moving the `loadQuestions`/`startListeningToQuestions` functions into shared logic as well.